arch/arm/boot/dts/overlays/revpi-flat-overlay.dts

```diff
@@ -111,30 +111,12 @@
 	fragment@1 {
 		target = <&gpio>;
 		__overlay__ {
-			i2c0_gpio44: i2c0_gpio44 {
-				brcm,pins = <44 45>;
-				brcm,function = <BCM2835_FSEL_ALT1>;
-			};
-			i2c1_gpio2: i2c1_gpio2 {
-				brcm,pins = <2 3>;
-				brcm,function = <BCM2835_FSEL_ALT0>;
-			};
-			spi0_cs_pins: spi0_cs_pins {
+			spi0_cs_pins {
```

Comment:
Removing this label will make the line 220 reverence the wrong pinmux
`pinctrl-0 = <&spi0_pins &spi0_cs_pins>;`
I guess it will use the following pinmux from arch/arm/boot/dts/bcm2710-rpi-cm3.dts:
```
	 spi0_cs_pins: spi0_cs_pins {
		brcm,pins = <8 7>;
		brcm,function = <1>; /* output */
	}; 
```

Reply:
Removing the label is not an issue (the existing spi0_cs_pins node is simply overridden) but the dts file in the commit message was indeed wrong. I fixed it in the new version